test/unit/test-custom-element.js

```diff
@@ -1584,7 +1584,8 @@ describes.realWin('CustomElement', {amp: true}, (env) => {
             .once();
 
           const promise = element.ensureLoaded(parentPriority);
-          await element.buildInternal();
+          await resource.build();
+          await resource.whenBuilt();
```

Comment:
is whenBuilt different from `resource.build()`?

Reply:
Yes. Unfortunately it is. This is in a nutshell exactly why the bug happened. This is just a quick fix. Our big-fix options:

1. Reimplement `Resource.whenBuild` and state.
2. Transition some elements to V1 faster. V1 do not have that problem by definition.